docs/processors/parallel-processors.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Parallel Processors'
+sidebar_position: 3
+---
+
+Conduit allows you to configure a processor to process records in parallel. This
```

Comment:
Maybe we actually mention the parameter.
So something like.
"... to process records in parallel using the `workers` parameter." 

The second sentence feels a little redundant. It is basically just defining parallelism.